ENH: add support for pd.Index for column kwarg in geodataframe.plot() (#3463)
geopandas/geopandas: CHANGELOG.md

  instead of its current behaviour of assuming EPSG. In the event the spiatal_ref_sys
  table is not present, or the SRID is not present, `read_postgis` will fallback
  on assuming EPSG CRS authority. (#3329)

> - `plot_dataframe` `column` kwarg will support `pd.Index` to colour based on the
>    geodataframe index (i.e. ``GeoDataFrame.plot(column=GeoDataFrame.index)``). (#3463)

Reviewer: ```suggestion
-  `GeoDataFrame.plot` now supports `pd.Index` as an input for the `column` keyword (#3463).
```

This is probably a bit better wording. Also, please move it all the way up to file it under (upcoming) version 1.1.0.

Author: Whoops, yes, put that in the wrong spot

Re-worded as suggested and moved to 1.1.0
